news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,171 @@
+
+To do so, he generally compares two installs, one on the branch and one with the PR, performs the same actions on each one and checks that the result is strictly identical. He can also use some parts of the test campaign usually used to test build releases
+
+### What happens next ?
+
+As for bug fix, if the tester finds any bug, he will comment the PR with all useful indicators to help the author to fix the bug.
+If he/she notices a missing behavior not specified (for example, a case where, in his opinion, an error should be displayed) he/she will ping the Product team and / or the maintainers to discuss this particular point.
+
+And if everything is fine, he changes the label to “QA Approved”, the PR is merged in the branch and will be available on PrestaShop in the next build of the branch !
+
+As for bug fix, these tests are tracked on an internal tool. The tester might also update some of the test campaigns to add or modify test cases related to this improvement.
+
+# IV. Migration Pull Request
+
+Migration Pull Request are quite different to other ones : the purpose of these PR is to move from a Legacy framework to a Symfony framework, while keeping the same functionalities (and some minor improvements).
+
+This makes the QA work on these Pull Request quite longer and harder than on other PR.
+
+### Requirements
+
+The main difference here is that the issue linked to the PR might be just an Epic listing all the items that should be migrated.
+
+For the “How to Test”, it’s sometimes only “Everything should work as in Legacy”. If there is a specification linked to this page, it should be added to the “How to Test”, to help the tester check everything. If the migrated page is “hidden”, the “How to Test” must also contain the URL to access it.
+
+The other requirements are the same.
+
+### Test of a Migration Pull Request
+
+The test of a Migration PR is quite long : The tester must compare the Legacy page and Migrated page, and verify that there are no errors or missing information, missing link, missing button... on the Migrated page, with every configuration he thinks of (multi-currencies, multi-languages, multistore, with different parameters).
+
+If there is a specification, he will follow every point and check it’s working fine. If there is not, he must find every detail by himself.
+
+During this run the tester might use the test campaign created for build releases, and also update these same tests if necessary.
+That’s why a test on a Migration PR takes more time than a “classic” one (sometimes several days).
+
+# Conclusion
+
+QA's work on Pull Request can be really time consuming, but it allows PrestaShop to maintain and even improve the quality of the solution.
+You can help us by submitting Pull Requests and issues as complete as possible, it will facilitate and speed up the processing of your Pull Request !
```

Comment:
```suggestion
Make sure that the Pull Requests and issues you submit are as thoroughly described as possible – it will help processing them quickly and seamlessly!
```